crowbar_framework/db/migrate/20150312081459_create_proposals.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProposals < ActiveRecord::Migration
+  def change
+    create_table :proposals do |t|
+      t.string :barclamp, :null => false
+      t.string :name, :null => false
+      t.text :properties
```

Comment:
And on top of that deployment and attributes is already handled more or less differntly, even when it is displayed on a single page :)